great_expectations/rule_based_profiler/profiler.py

```diff
@@ -22,11 +23,14 @@
 
 
 class Profiler:
-    """Profiler object serves to profile, or automatically evaluate a set of rules, upon a given
-    batch / multiple batches of data"""
+    """
+    Profiler object serves to profile, or automatically evaluate a set of rules, upon a given
+    batch / multiple batches of data.
+    """
 
     def __init__(
         self,
+        validator: Validator,
```

Comment:
As discussed, we may want to remove the need (and potentially the ability) to pass in a Validator to a Profiler in a coming PR.